src/assignment/views/AssignmentResponseEdit/AssignmentResponseEdit.tsx

```diff
@@ -124,8 +124,8 @@ const AssignmentResponseEdit: FunctionComponent<
 		}
 	);
 
-	const pastDeadline = useMemo(
-		() => assignment?.deadline_at && isPast(new Date(assignment.deadline_at)),
+	const pastDeadline: boolean = useMemo(
+		() => (assignment?.deadline_at && isPast(new Date(assignment.deadline_at))) || false,
 		[assignment]
 	);
```

Comment:
Align with implementation in [`src/assignment/hooks/assignment-pupil-tabs.tsx`](https://github.com/viaacode/avo2-client/pull/1402/files#diff-a90f9148badf347235e9ccea843f8e7daec08786167408c9cac290c1787725f1) (split to hook/helper/...)